src/commands/autocomplete.ts

```diff
@@ -15,15 +15,15 @@ export const xrefStarterAutoComplete: CompletionItemProvider = {
         token: CancellationToken,
         context: CompletionContext): ProviderResult<CompletionList<CompletionItem> | CompletionItem[]> => {
 
-        const range = document.getWordRangeAtPosition(position, /[<(](xref):/si);
+        const range = document.getWordRangeAtPosition(position, /<xref:/si);
```

Comment:
1. Are the parenthesis around xref required? It seemed not.
2. I realize this removes the code action for `[](xref:)` but there wasn't a straightforward way to deal with the parentheses instead of the angle brackets. Because now with the code action, the <Xref:> is always replaced (for the purpose of letting the user type <xref:> to bring up the insert API command, but then selecting custom text), and `skipBrackets` refers specifically to angle brackets.